.changeset/cold-clocks-hear.md

```diff
@@ -2,7 +2,7 @@
 '@guardian/source-react-components-development-kitchen': major
 ---
 
-Refactor ExpandingWrapper so it can receive an optional theme. This will require
+Refactor `ExpandingWrapper` so it can receive an optional theme. This will require
 consumers to update their calls, but it should be compatible with the existing
```

Comment:
is it possible to explain how people should do it? just because the devkit api is so unstable, the clearer the release notes can be the easier it will make upgrading.

Reply:
They should convert their `cssOverrides` to specific colour values for each of the relevant members, not sure how to best express that.

Reply:
lovely